apps/frontend/src/libs/enums/svg-icons.enum.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/// <reference types="vite-plugin-svgr/client" />
+
+import accessIcon from "../../assets/images/icons/access.svg?react";
+import analyticsIcon from "../../assets/images/icons/analytics.svg?react";
+import contributorsIcon from "../../assets/images/icons/contributors.svg?react";
+import projectIcon from "../../assets/images/icons/project.svg?react";
+
+const Icons = {
```

Comment:
it's a map data structure
https://whatislove.dev/articles/js-and-everyday-data-structures/